config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
```

Comment:
- [ OPTIONAL ] I think it would be great if you remove this two lines and add in the [ ] square brackets :show and :index so kindly fix them 🔥🤩

Reply:
Thank you @Reem-lab